src/App.css

```diff
@@ -137,7 +137,7 @@ body {
   width: 13%;
```

Comment:
- [ ] [OPTIONAL] Kindly make the design responsive, as responsitivity is crucial in today's web development world. 😀 


![mobile](https://user-images.githubusercontent.com/41163571/233631825-5e961961-df0b-4c7a-95a2-491f9d84562b.gif)